modules/interaction/src/blaze/interaction/history/instance.clj

```diff
@@ -14,7 +14,8 @@
     [reitit.core :as reitit]
     [ring.middleware.params :refer [wrap-params]]
     [ring.util.response :as ring]
-    [taoensso.timbre :as log]))
+    [taoensso.timbre :as log]
+    [clojure.string :as str]))
```

Comment:
Please order the requires lexicographically. Unfortunately Cursive is not doing this automatically in the moment.